Description: Do not strip the SoundStretch binary by default.
 If we use DEB_BUILD_OPTIONS=nostrip the SoundStretch binary is still stripped.
 .
 Correct this behavior by eliminating the -s flags.
Bug-Debian: http://bugs.debian.org/438038
Forwarded: yes
Author: Miguel Colon <debian.micove@gmail.com>
Last-Update: 2011-02-25
--- soundtouch-1.5.0.orig/source/SoundStretch/Makefile.am
+++ soundtouch-1.5.0/source/SoundStretch/Makefile.am
@@ -44,7 +44,7 @@ soundstretch_SOURCES=main.cpp RunParamet
 soundstretch_LDADD=../SoundTouch/libSoundTouch.la -lm
 
 ## linker flags
-soundstretch_LDFLAGS=-s
+#soundstretch_LDFLAGS=-s
 
 ## additional compiler flags
 soundstretch_CXXFLAGS=-O3
